daemon/assets.py

```diff
@@ -16,16 +16,8 @@
 import utils
 
 
-def get_res_file(data, find_closest_with_url: bool = False):
+def get_res_file(data):
   """Returns closest resolution that current asset can offer.
-  
```

Comment:
We can't Remove find_closest_url:
Its an important functionality, let's say the user sets his default res to 4k, but the asset has only .5k-2k resolution, then it finds that 2k is closest. Potentially also asset could only have 1k/2k, and the user a lower setting. If this functionality wasn't used right now, we're doing something very wrong.

Reply:
also, the setting is off e.g. for original file, or some other specific cases.

Reply:
Before and after this PR the function searches for the closest resolution by  default. We just did not use the parameter to turn that functionality off.